radis/lbl/base.py

```diff
@@ -3202,6 +3209,26 @@ def _cutoff_linestrength(self, cutoff=None):
 
             error = df.S[b].sum() / df.S.sum() * 100
 
+            if cutoff_error is not None:
+                if cutoff_error < error:
+                    # Remove additional lines such that cutoff error is less than user-inputted value
+                    while cutoff_error < error:
+                        cutoff -= 0.0001
+                        b = df.S <= cutoff
+                        error = df.S[b].sum() / df.S.sum() * 100
```

Comment:
what about : 
1 . we sort (a copy of) the lines by intensity intensity
2. we compute the cumulated intensity on the sorted array  [last value is the total sum]
3.  we find the index in the sorted array so that cumulated-intensity is ERROR_fraction * the total

Reply:
Yes, this seems to be a much better option. I'll work on this.

Reply:
Please let me know. It will be important to also add tests.

Reply:
Hello @sagarchotalia ,did you try the approach I suggested above ?